services/api/service.go

```diff
@@ -72,6 +72,7 @@ var (
 	numActiveValidatorProcessors = cli.GetEnvInt("NUM_ACTIVE_VALIDATOR_PROCESSORS", 10)
 	numValidatorRegProcessors    = cli.GetEnvInt("NUM_VALIDATOR_REG_PROCESSORS", 10)
 	timeoutGetPayloadRetryMs     = cli.GetEnvInt("GETPAYLOAD_RETRY_TIMEOUT_MS", 100)
+	timeoutGetPayloadResponseMs  = cli.GetEnvInt("GETPAYLOAD_RESPONSE_TIMEOUT_MS", 1000)
```

Comment:
(for later) I'd name this "GETPAYLOAD_DELAY_MS"

Reply:
good catch, done